File: Homework/Homework1/Palindrome.py
def is_palindrome(s):
    s = s.lower()
    s = ''.join(char for char in s if char.isalnum())

    reverse = ""
    for i in range(len(s) - 1, -1, -1):
        reverse += s[i]

    return s == reverse

if __name__ == "__main__":
    test_string = "racecar"
    print(is_palindrome(test_string))

File: Homework/Homework1/Palindrome.java
package hw1;

public class Palindrome {
	
	
	public static boolean isPalindrome(String s) {
		
		s.toLowerCase();
		s.replaceAll("[^a-zA-Z0-9]", "");
		
		String reverse = "";
		
		for (int i = s.length() - 1; i >= 0; --i) {
			
			reverse = reverse + s.charAt(i);
			
		}
		
		if (s.equals(reverse)) {
			return true;
		}
		
		
		
		return false;
	}
	
	
	public static void main (String[] args) {
		String s = "racecar";
		System.out.print(isPalindrome(s));
		
	}
	
	
	

}
